src/vulnerabilities/vulnerability-report.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
potentially also move to spacecat-shared to that the UI may reuse the same type definitions